src/pages/project/project.less

```diff
@@ -74,6 +74,13 @@
 
     &.source {
       opacity: 0.75;
+      .shim .indicator {
+        width: 50%;
+        height: 160/220 * 50%;
```

Comment:
Can you move them up into a variable at the top of the file? If we're going to be doing inline calculations, we should be able to trace why we're doing them.